src/kernels/jupyter/serverSelector.ts

```diff
@@ -54,11 +51,7 @@ export type SelectJupyterUriCommandSource =
     | 'prompt';
 @injectable()
 export class JupyterServerSelector {
-    private readonly localLabel = `$(zap) ${DataScience.jupyterSelectURINoneLabel()}`;
-    private readonly newLabel = `$(server) ${DataScience.jupyterSelectURINewLabel()}`;
-    private readonly remoteLabel = `$(server) ${DataScience.jupyterSelectURIRemoteLabel()}`;
```

Comment:
Are these not needed anymore?

Reply:
No they shouldn't be. Maybe. I think we might need the 'None' label.